app/crud.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+
+def get_city(db: Session, city_id: int):
+    return db.query(models.City).filter(models.City.id == city_id).first()

```

Comment:
Consider adding error handling for the case where the city with the given `city_id` does not exist. This will make the function more robust and prevent potential issues when the result is `None`.